cli/fmt.rs

```diff
@@ -37,15 +37,23 @@ fn is_supported(path: &Path) -> bool {
 
 fn get_config() -> dprint::configuration::Configuration {
   use dprint::configuration::*;
-  ConfigurationBuilder::new()
-    .line_width(80)
-    .indent_width(2)
-    .next_control_flow_position(NextControlFlowPosition::SameLine)
-    .binary_expression_operator_position(OperatorPosition::SameLine)
-    .brace_position(BracePosition::SameLine) // default is NextLineIfHanging
-    .comment_line_force_space_after_slashes(false)
-    .construct_signature_space_after_new_keyword(true)
-    .build()
+  ConfigurationBuilder::new().prettier().build()
```

Comment:
Note that this now uses the Prettier 2.0 config: https://prettier.io/blog/2020/03/21/2.0.0.html

The exception is that it uses what is essentially `--trailing-comma all` instead of `es5`. Is that ok? If not, I can easily implement an ES5 setting in Dprint... I just thought it probably wasn't worth it given that Prettier will eventually make the jump to "all" once people stop writing JS for ES5 environments.

Additional note: it's slightly "better" than Prettier's `all` for TypeScript, as it also gets trailing commas in type parameters, but not type arguments since TS does not support that yet.

Reply:
sounds good to me